components/ILIAS/Mail/classes/Object/class.ilMailValueObject.php

```diff
@@ -23,28 +23,45 @@
  */
 class ilMailValueObject
 {
+    private string $recipients;
+    private string $recipientsCC;
+    private string $recipientsBCC;
+    private string $subject;
+    private string $body;
     /** @var string[] */
-    private readonly array $attachments;
+    private array $attachments;
+    private bool $usePlaceholders;
+    private bool $saveInSentBox;
+    private string $from;
 
     /**
      * @param string[] $attachments
      * @throws InvalidArgumentException
      */
     public function __construct(
-        private readonly string $from,
-        private readonly string $recipients,
-        private readonly string $recipientsCC,
-        private readonly string $recipientsBCC,
-        private readonly string $subject,
-        private readonly string $body,
+        string $from,
+        string $recipients,
+        string $recipientsCC,
+        string $recipientsBCC,
+        string $subject,
+        string $body,
         array $attachments,
-        private readonly bool $usePlaceholders = false,
-        private readonly bool $saveInSentBox = false
+        bool $usePlaceholders = false,
+        bool $saveInSentBox = false
```

Comment:
With the changes coming soon I will set 'readonly'  above the constructor for the member variables instead of inside of the constructor for regularity purposes.